tools/roslaunch/test/unit/test_xmlloader.py

```diff
@@ -202,9 +199,9 @@ def test_params(self):
         self.assertEquals(2, param_d['/someinteger2'])
         self.assertAlmostEquals(3.14159, param_d['/somefloat1'], 2)
         self.assertAlmostEquals(5.0, param_d['/somefloat2'], 1)
-        self.assertEquals("a child namespace parameter 1", param_d['/wg/wgchildparam'], p.value)
-        self.assertEquals("a child namespace parameter 2", param_d['/wg2/wg2childparam1'], p.value)
-        self.assertEquals("a child namespace parameter 3", param_d['/wg2/wg2childparam2'], p.value)
+        self.assertEquals("a child namespace parameter 1", param_d['/wg/wgchildparam'])
+        self.assertEquals("a child namespace parameter 2", param_d['/wg2/wg2childparam1'])
+        self.assertEquals("a child namespace parameter 3", param_d['/wg2/wg2childparam2'])
```

Comment:
Why is the third argument removed here?

Reply:
Because ``p`` does not exist anymore. Actually I don't really know, how this could have worked in the first place.

Reply:
Ok it worked, because loop-variables are not bound to the loops-scope in python ([see here](https://stackoverflow.com/questions/3611760/scoping-in-python-for-loops)). Anyway, I'd argue, that the asserts are sufficient the way they are.